lib/src/status.rs

```diff
@@ -305,12 +306,24 @@ pub(crate) async fn status(opts: super::cli::StatusOpts) -> Result<()> {
     let legacy_opt = if opts.json {
         OutputFormat::Json
     } else {
-        OutputFormat::Yaml
+        if  std::io::stdout().is_terminal() {
+            OutputFormat::HumanReadable
+        } else {
+            OutputFormat::Yaml
+        }
     };
     let format = opts.format.unwrap_or(legacy_opt);
     match format {
         OutputFormat::Json => serde_json::to_writer(&mut out, &host).map_err(anyhow::Error::new),
         OutputFormat::Yaml => serde_yaml::to_writer(&mut out, &host).map_err(anyhow::Error::new),
+        OutputFormat::HumanReadable => {
```

Comment:
Looks like a sane start to me, but we'll probably want to split this into a new function